src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.ILLink.targets

```diff
@@ -88,6 +89,12 @@ Copyright (c) .NET Foundation. All rights reserved.
       </ManagedAssemblyToLink>
     </ItemGroup>
 
+    <!-- Map public properties to the task options -->
+    <PropertyGroup>
+      <_TrimmerLinkSymbols Condition=" '$(TrimmerRemoveSymbols)' == 'true' ">false</_TrimmerLinkSymbols>
+      <_TrimmerLinkSymbols Condition=" '$(TrimmerRemoveSymbols)' != 'true' ">true</_TrimmerLinkSymbols>
```

Comment:
Do we need the `_TrimmerLinkSymbols` property anymore?

Reply:
Since the naming is inverted, it is probably easier to read this way. Unless we wanted to invert the ILLink Task option.

Reply:
It's necessary because the existing task option has the opposite polarity of the public property. (I don't think you can pass `="! $(TrimmerRemoveSymbols)"`). We could remove this by changing the task option (and maybe the command-line arguments).